mods/common/languages/en.ftl

```diff
@@ -64,8 +64,9 @@ notification-invalid-configuration-command = Invalid configuration command.
 notification-admin-option = Only the host can set that option.
 notification-error-number-teams = Number of teams could not be parsed: { $raw }
 notification-admin-kick = Only the host can kick players.
+notification-kick-self = The host is not allowed to kick himself.
```

Comment:
This is still addressing a male audience.

Reply:
It's addressing everyone, regardless of gender

Reply:
English does not have the generic masculine as for example in German. 
> You use themselves instead of 'himself or herself' to refer back to the person who is the subject of a [sentence](https://www.collinsdictionary.com/dictionary/english/sentence) without [saying](https://www.collinsdictionary.com/dictionary/english/saying) whether it is a man or a woman.

Source: https://www.collinsdictionary.com/dictionary/english/themselves 3.

Reply:
sure, this is better than `themself` suggestion

Reply:
fwiw english has generic masculine, but often english users use linguistic strategies to make it non-gender specific or invent new words

Reply:
Bruh.